### ISSUE TITLE
get stable-compatible

### ISSUE THREAD

**Issue by blaenk** [cobalt-org/liquid-rust#1]
Hi! You might be interested in going stable-compatible. This means that the project can't use any unstable language features. This is currently using:

``` rust
#![feature(box_syntax)]
#![feature(unboxed_closures)]
#![feature(plugin)]
#![feature(test)]
#![feature(collections)]
#![feature(core)]
```

You can easily identify which parts of your code require which features by removing those attributes.

The box syntax feature is an easy one to do away with by replacing `box someexpr` with `Box::new(someexpr)`.

The `plugin` feature is mostly required due to your use of `regex_macros`, a very nice package indeed but one which requires syntax extension support, which stable doesn't. This is easily remedied by changing `regex!("pattern")` to `Regex::new("pattern")` and checking for potential failure, or unwrapping if the pattern isn't dynamically generated.

The test feature is usually needed for benchmarking purposes.

The rest you can figure out by removing the given attribute and seeing what generates an error.

Common strategies for going stable include:

* introducing cargo features, such as one called "nightly" and using the `cfg(feature = "nightly")` attribute to conditionally compile functionality that requires nightly features
* ripping out implementations of nightly features from the rust std and embedding them into your code base directly

**Comment by tak1n** [comment]
Thx for opening this issue, we are currently looking into this. :+1:

**Comment by johannhof** [comment]
Thanks, closed by 09e546f946bd0f5e95f0634fcd161ddb06bb3deb

**Comment by blaenk** [comment]
Great work! :+1:
